packages/web/src/dev/ValidatorsListStyles.tsx

```diff
@@ -220,4 +229,46 @@ export const styles = StyleSheet.create({
   barKo: {
     backgroundColor: colors.red,
   },
+
+  // Checkmark
+  checkmark: {
+    display: 'inline-block',
+    width: 14,
+    height: 14,
+    lineHeight: 14,
+    backgroundColor: colors.white,
+    borderRadius: '50%',
+    textAlign: 'center',
+    marginLeft: 6,
+    position: 'relative',
+  } as any,
+
+  // Tooltip
+  tooltip: {
+    fontWeight: '300',
+    fontSize: 14,
+    backgroundColor: '#585c60',
```

Comment:
nit: would prefer to use color enum here

Reply:
It is not a color of the palette